android/jni/com/mapswithme/maps/editor/Editor.cpp

```diff
@@ -356,4 +356,10 @@ Java_com_mapswithme_maps_editor_Editor_nativeCreateNote(JNIEnv * env, jclass cla
   Editor::Instance().CreateNote(MercatorBounds::FromLatLon(lat, lon), jni::ToNativeString(env, text));
 }
 
+// static boolean nativeIsHouseValid(String houseNumber);
+JNIEXPORT jboolean JNICALL
+Java_com_mapswithme_maps_editor_Editor_nativeIsHouseValid(JNIEnv * env, jclass clazz, jstring houseNumber)
```

Comment:
Это означает, что аргументы нигде ниже не используются. Очень удобно читать такую сигнатуру. Еще можно комментировать аргументы. В плюсах компилер дает ворнинги, если остаются не используемые аргументы.

> On Apr 7, 2016, at 13:03, Dmitry Yunitsky <notifications@github.com> wrote:
> 
> In android/jni/com/mapswithme/maps/editor/Editor.cpp <https://github.com/mapsme/omim/pull/2810#discussion_r58847721>:
> 
> > @@ -356,4 +356,10 @@ Java_com_mapswithme_maps_editor_Editor_nativeCreateNote(JNIEnv * env, jclass cla
> >    Editor::Instance().CreateNote(MercatorBounds::FromLatLon(lat, lon), jni::ToNativeString(env, text));
> >  }
> >  
> > +// static boolean nativeIsHouseValid(String houseNumber);
> > +JNIEXPORT jboolean JNICALL
> > +Java_com_mapswithme_maps_editor_Editor_nativeIsHouseValid(JNIEnv * env, jclass clazz, jstring houseNumber)
> Не нравится мне без названий аргументов:)
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2810/files/94e4930e51d14e01f30778e20b0549e6286b8fed#r58847721>